### PR TITLE
chore: improve readme

### PR DESCRIPTION
**Pull request by NeverHappened**
Implements [this task](https://p2pvalidator.atlassian.net/browse/NTRN-116).

ALSO removes two unnecessary variables:
- RELAYER_NEUTRON_CHAIN_CHAIN_PREFIX (unused)
- RELAYER_NEUTRON_CHAIN_ACCOUNT_PREFIX (can be taken from neutron chain)

### CODE REVIEW

**Comment by sotnikov-s** [comment]
could you please do a little out of scope thingy? the RELAYER_NEUTRON_CHAIN_CHAIN_PREFIX is not used at all and really looks redundant since it's actually a neutron's relayer. we set the prefix here: https://github.com/neutron-org/neutron-query-relayer/blob/main/internal/app/app.go#L55

**Comment by sotnikov-s** [comment]
> could you please do a little out of scope thingy? the RELAYER_NEUTRON_CHAIN_CHAIN_PREFIX is not used at all and really looks redundant since it's actually a neutron's relayer. we set the prefix here: https://github.com/neutron-org/neutron-query-relayer/blob/main/internal/app/app.go#L55

this one is also never used: RELAYER_NEUTRON_CHAIN_ACCOUNT_PREFIX

**Comment by sotnikov-s** [comment]
also please fix these misprints:
- **target** chain provider output format -> **neutron** chain provider output format;
- target chain **connetcion** ID -> target chain **connection** ID.
Screen Shot 2022-09-19 at 11 42 22

File: internal/relay/config.go [hunk 1]
```diff
@@ -10,12 +10,12 @@ import (
 	"github.com/neutron-org/neutron-query-relayer/internal/config"
 )
 
-func GetNeutronChain(logger *zap.Logger, cfg *config.NeutronChainConfig) (*relayer.Chain, error) {
+func GetNeutronChain(logger *zap.Logger, cfg *config.NeutronChainConfig, accountPrefix string) (*relayer.Chain, error) {
```
Comment (swelf19) [inline internal/relay/config.go:13]: I dont think it makes sense to pass the prefix as variable, you can use `neutronapp.Bech32MainPrefix` as is, during  `cosmos.CosmosProviderConfig` instantiating

Comment (NeverHappened) [reply internal/relay/config.go:13]: Yeah, okay, thought about that, but wasn't sure

File: README.md [hunk 2]
```diff
@@ -1,23 +1,56 @@
-# Description
+Interchain query relayer implementation for [Neutron](https://github.com/neutron-org/neutron).
 
-Interchain query relayer implementation for Neutron
+More on relayer's configuration and inner can be read in [neutron-docs](https://github.com/neutron-org/neutron-docs/blob/main/docs/relaying/icq-relayer-guide.md)
```
Comment (sotnikov-s) [inline README.md:3]: yes, it's better to use the link to the github pages: https://neutron-org.github.io/neutron-docs/relaying/icq-relayer

**Review by sotnikov-s — APPROVED** [review]

**Review by swelf19 — APPROVED** [review]

**Pull request merged**